spec/facility_spec.rb

```diff
@@ -1,26 +1,211 @@
+      expect(can_road_test).to eq(false)
+      @registrant.license_data[:license] = true
+      can_renew = @facility1.renew_drivers_license(@registrant)
+      expect(can_renew).to eq(false)
+    end
+    it "can provide services after they are added" do
+      @facility1.add_service("Written Test")
+      expect(@facility1.services.include?("Written Test")).to eq(true)
+      can_register = @facility1.administer_written_test(@registrant)
+      expect(can_register).to eq(true)
+      @facility1.add_service("Road Test")
+      can_road_test = @facility1.administer_road_test(@registrant)
+      expect(can_road_test).to eq(true)
+      @facility1.add_service("Renew License")
+      can_renew = @facility1.renew_drivers_license(@registrant)
+      expect(can_renew).to eq(true)
+    end
+    it "does not provide license services to registrants under 16" do
+      @facility1.add_service("Written Test")
+      @facility1.add_service("Road Test")
+      registrant = Registrant.new({
+        name: "Youngun",
+        age: 15,
+        permit: true
+      })
+      can_do_written_test = @facility1.administer_written_test(registrant)
+      expect(can_do_written_test).to eq(false)
+      can_get_license = @facility1.administer_road_test(registrant)
+      expect(can_get_license).to eq(false)
+    end
+    it "no road tests for people who haven't completed the written test" do
+      @facility1.add_service("Written Test")
+      @facility1.add_service("Road Test")
+      @has_license = @facility1.administer_road_test(@registrant)
+      expect(@has_license).to eq(false)
+      expect(@registrant.license_data[:license]).to eq(false)
+    end
+    it "gives licenses to those who complete the road test" do
+      @facility1.add_service("Written Test")
+      @facility1.add_service("Road Test")
+      @facility1.administer_written_test(@registrant)
+      has_license = @facility1.administer_road_test(@registrant)
+      expect(has_license).to eq(true)
+      expect(@registrant.license_data[:license]).to eq(true)
+    end
+    it "does not renew licenses to those without a license" do
+      @facility1.add_service("Written Test")
+      @facility1.add_service("Road Test")
+      @facility1.add_service("Renew License")
+      @facility1.administer_written_test(@registrant)
+      can_renew = @facility1.renew_drivers_license(@registrant)
+      expect(can_renew).to eq(false)
+      expect(@registrant.license_data[:renewed]).to eq(false)
+    end
+    it "can renew licenses to those with a license" do
+      @facility1.add_service("Written Test")
+      @facility1.add_service("Road Test")
+      @facility1.add_service("Renew License")
+      @facility1.administer_written_test(@registrant)
+      @facility1.administer_road_test(@registrant)
+      can_renew = @facility1.renew_drivers_license(@registrant)
+      expect(can_renew).to eq(true)
+      expect(@registrant.license_data[:renewed]).to eq(true)
+    end
```

Comment:
Great job testing all of these conditions! I know it's a lot so we had a lot to test. 

Again, go ahead and use white spaces to help organize between blocks of code and even to organize within an it block. I like to put a white spaace between my setup, my execution, and my assertion so I can visually see those sections easily.